.circleci/config.yml

```diff
@@ -58,10 +59,15 @@ jobs:
       - run:
           name: Run Unit Tests
           command: |
+            set -x
+            pip install -r requirements-coveralls.txt
```

Comment:
May want to rename the requirements file as the application used is no longer coveralls.  requirements-codecoverage.txt?

Reply:
Renamed 'requirements-coveralls.txt' as 'requirements-codecoverage.txt'